### DISCUSSION TITLE
Are example files cited in tutorial available?

### DISCUSSION THREAD

**Discussion by bertiewooster** [rdkit/rdkit#5799] [Q&A] [answered]
[Getting Started with the RDKit in Python](https://www.rdkit.org/docs/GettingStartedInPython.html#degen) cites several files which the reader is apparently supposed to have access to, for example [`data/input.mol`](https://www.rdkit.org/docs/GettingStartedInPython.html#reading-single-molecules), [`data/5ht3ligs.sdf`](https://www.rdkit.org/docs/GettingStartedInPython.html#reading-sets-of-molecules), [`data/chiral.mol`](https://www.rdkit.org/docs/GettingStartedInPython.html#writing-molecules), etc. Is there a collection of those files available? Thanks!

**Comment by greglandrum** [comment, marked as answer]
These files are in the directory `Docs/Book/data`: https://github.com/rdkit/rdkit/tree/master/Docs/Book/data

**Reply by bertiewooster** [reply]
Thanks! I was able to download the files, though one at a time and with several manual steps per file. Short of telling learners to clone the entire repository, I wonder if there's a way to create a .zip archive (using CI) that would let learners download all the files at once?
